test/amperize.test.js

```diff
@@ -676,30 +676,6 @@ describe('Amperize', function () {
                 done();
             });
         });
-
-        it('can handle timeout errors', function (done) {
```

Comment:
This test seemed very reliant on the now removed 'Request' library. Just removing it now that it uses Axios.